scripts/install-vsix-dependencies.js

```diff
@@ -7,16 +7,23 @@ const shell = require('shelljs');
 // Installs a list of extensions passed on the command line
 var version = process.env.CODE_VERSION;
 
+console.log('### install-vsix-dependencies start');
 console.log('CODE_VERSION: ' + version);
 
 var isInsiders = version === 'insiders';
 
+const contentOfVscodeTest = fs.readdirSync(`${process.cwd()}/.vscode-test`);
+let vscodeDownloadDir = contentOfVscodeTest.find(file =>
+  file.startsWith('vscode-')
+);
+
+console.log('### where is VScode', { vscodeDownloadDir });
 // VSCode no longer downloads to a single directory name like 'stable'. The folder
 // name is dynamic base on the version number, so lets just use the first folder in .vscode-test dir
 // as the assumed place where vscode is extracted
 const testRunFolder = path.join(
   '.vscode-test',
-  isInsiders ? 'insiders' : fs.readdirSync(`${process.cwd()}/.vscode-test`)[0]
+  isInsiders ? 'insiders' : vscodeDownloadDir
```

Comment:
I'll remove the above comment.  This wasn't true for multiple runs of the int tests and ofter resulted in failure locally.